docs/content/en/docs-dev/user-guide/event-watcher.md

```diff
@@ -18,8 +18,7 @@ While it empowers you to build pretty versatile workflows, the canonical use cas
 This guide walks you through configuring Event watcher and how to push an Event.
 
 ## Prerequisites
-If you want to use EventWatcher configuration files under the `.pipe/` directory, before we get into configuring EventWatcher, be sure to configure Piped. See [here](/docs/user-guide/managing-piped/configuring-event-watcher/) for more details.
-In other cases, there is no prerequisite. Let's move on to Usage.
+Before we get into configuring EventWatcher, be sure to configure Piped. See [here](/docs/user-guide/managing-piped/configuring-event-watcher/) for more details.
```

Comment:
This is only true in case of using `.pipe/` pattern, isn't it?

Reply:
Sorry, we need to configure `repoID` hence fixed like this.